cpp/scripts/run-clang-compile.py

```diff
@@ -0,0 +1,320 @@
+# Copyright (c) 2020-2021, NVIDIA CORPORATION.
```

Comment:
Just a reminder to update copyrights.

Reply:
I've found more than 300 files in RAFT not passing the copyright checker, I'll probably file a separate PR for this, although I'll update the copyrights of the files in this particular PR as well.